mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -9,16 +9,19 @@ CLASS avx net/minecraft/entity/player/PlayerEntity
 		METHOD a getText ()Ljm;
 	FIELD bA container Layb;
 	FIELD bB hungerManager Laxx;
+	FIELD bF experiencePickUpDelay I
 	FIELD b SIZES Ljava/util/Map;
 	FIELD bM isInWater Z
 	FIELD bN abilities Lavu;
-	FIELD bO experience I
-	FIELD bP experienceLevel I
-	FIELD bQ experienceLevelProgress F
+	FIELD bO xpLevel I
+	FIELD bP xpTotal I
+	FIELD bQ xpProgress F
```

Comment:
Blayyke pointed out, it might be better to use different names, so mods that used the previous names won't compile with the swapped names.
Also the corresponding nbt fields are `XpLevel`, `XpTotal` and `XpP`.

Reply:
All the other experience-related names are `experience` though. They would have to all be changed for consistency.

Reply:
Yes.

Reply:
I wanted to get more input first, before changing everything. I kinda liked the methods being `experience`.

Reply:
I think I'm going to go with
`experience` -> `experienceLevel`
`experienceLevel` -> `experienceTotal`
now. It's going to break mods that use both, or only the former `experienceLevel`